newIDE/app/src/UI/SemiControlledAutoComplete.js

```diff
@@ -181,21 +177,20 @@ const getOptionLabel = (option: Option, value: string): string =>
 export default React.forwardRef<Props, SemiControlledAutoCompleteInterface>(
   function SemiControlledAutoComplete(props: Props, ref) {
     const input = React.useRef((null: ?HTMLInputElement));
-    const [state, setState] = useState<State>({ inputValue: null });
-    const [isMenuOpen, setIsMenuOpen] = useState<boolean>(true);
+    const [inputValue, setInputValue] = useState((null: string | null));
+    const [isMenuOpen, setIsMenuOpen] = useState(true);
```

Comment:
@4ian, this is the issue, this should basically mean open the menu when autocomplete is rendered (for the first time). I've fixed this by setting it false. The menu will only open up when autocomplete is focused.

Reply:
Sounds good 👍